samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -10,7 +10,7 @@ related:
 ---
 
 
-This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using the RabbitMQ direct routing topology. The direct routing topology is used by endpoints that are using NServiceBus Version 6. ServiceControl is built on NServiceBus Version 5 and can only use the conventional routing topology. The ServiceControl transport adapter is required to bridge both routing topologies.
+This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using the RabbitMQ direct routing topology, the default topology of endpoints using RabbitMQ on NServiceBus Version 6. However, since ServiceControl is built on NServiceBus Version 5 and can only use the conventional routing topology, a kind of "transport adapter" is required to bridge the routing topologies. 
```

Comment:
> This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using the RabbitMQ direct routing topology, the default topology of endpoints using RabbitMQ on NServiceBus Version 6.

@Particular/rabbitmq-transport-maintainers can you confirm that `DirectRoutingTopology` is the default in V6?
Reading [this](https://docs.particular.net/transports/rabbitmq/routing-topology#conventional-routing-topology) it's the other way round.